CONTRIBUTING.md

```diff
@@ -16,7 +17,27 @@ to existing libdefs.
 * [Writing tests](#writing-tests)
   * [Use `describe` and `it` blocks to limit scope](#use-describe-and-it-blocks-to-limit-scope)
 
-## Contributing to the definitions repository
+## Running libdef tests
+
+We write tests for libdefs. When you make changes to a libdef, you should make sure the tests still pass or add new tests for new functionality.
```

Comment:
`you should make sure the tests still pass or add new tests for new functionality ` -> `verify that the tests still pass, and add tests to cover new functionality `.

Reply:
Thanks. #firstPassesAtWords